src/mantine-hooks/src/use-fullscreen/use-fullscreen.ts

```diff
@@ -0,0 +1,82 @@
+import { useCallback, useRef, useState, useEffect } from 'react';
+
+function getFullscreenElement(): HTMLElement | null {
+  const fullscreenElement =
+    document.fullscreenElement ||
+    // @ts-expect-error
```

Comment:
It will be much cleaner to assign document with any type to variable instead of repeating @ts-expect-error each time:

```tsx
const _document = window.document as any;
```